### ISSUE TITLE
Is this license still an active project?

### ISSUE THREAD

**Issue by j-a-pope** [climate-strike/license#26]
I've just added the license to one of our projects, and made a PR here to add it to the Green List, but I notice that nothing else is added there and there seems to have been no project activity for a year. 

Are there are any plans to develop on and increase adoption of the license, or is it essentially a nice idea that didn't gain any traction?

**Comment by solegalli** [comment]
yep, same question here :)

**Comment by frnsys** [comment]
Hey, thanks for your interest in the project. We got sidetracked by the pandemic and haven't had a moment to return to this, but hopefully will be able to soon. The license itself is usable though there are some ambiguities around how it works with GPL that we haven't worked out yet.

I just merged your PR, thank you for submitting it.

**Comment by solegalli** [comment]
Is there anything we could do to support the project?

**Comment by frnsys** [comment]
Definitely! At this point the most important thing is adoption--so getting more devs to know about the project and use it for their own work. Help is very much appreciated. I'll find more time for this too once this current project I'm on starts winding down in a couple months.

**Comment by solegalli** [comment]
Great! It would be good to know what you learnt about enforcement. I already floated the idea to sklearn and other packages, and I got the feedback that they don't know how to enforce and also don't have the resource and the money to do so.

It would be good to know what you guys learned about it, so we can have those discussions. Maybe a small catch-up when you have more time?

**Comment by j-a-pope** [comment]
Good to hear that this is still a thing and that there are some plans :)

Adoption is going to be difficult to kickstart I think. 

I'm going to add it to more of our projects as we update them, I think having a greenlist that has more than one or two thing son it helps visually a lot. I'm also going to put together a blog post about the license, not sure that many people will read it, but every mention is something.

**Comment by PythonLinks** [comment]
I am working on building maps of organizations fighting climate change.  Maps will have pins, but the international map will have a list of international organizations, national maps will have a list of national organizations, etc. 

I could release the json under this license. 
The site will use the same software as 
GreenMaps.US
uses.

**Comment by j-a-pope** [comment]
> I am working on building maps of organizations fighting climate change. Maps will have pins, but the international map will have a list of international organizations, national maps will have a list of national organizations, etc.

Sounds like a really interesting project, please share a link when it launches.

**Comment by stebbi36** [comment]
> > I am working on building maps of organizations fighting climate change. Maps will have pins, but the international map will have a list of international organizations, national maps will have a list of national organizations, etc.
> 
> Sounds like a really interesting project, please share a link when it launches.

I want to make maps

**Comment by PythonLinks** [comment]
What maps do you want to make?
I could not reach you on facebook, as I am not willing to agree to their new terms of service, so you can reach me at 
https://mastodon.social/deck/@UncensoredNews

**Comment by stefanajons** [comment]
Jes

**Comment by stebbi36** [comment]
To make maps a friend sone and I will make it wear all whu are friends know, infarct I would like to make it avatar wourld
